docs/user/setting_up_your_development_environment.md

```diff
@@ -0,0 +1,78 @@
+#Setting up your development environment
```

Comment:
`# Setting up your development environment. `

There should be a space after every header sign for markdown. Same for the others.